src/backend/core/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 5.0.3 on 2024-03-25 22:58
+# Generated by Django 5.0.3 on 2024-04-10 09:32
```

Comment:
We had previously decided to rewrite all migrations to stick to 0001_initial until release. But I think Antoine mentioned he wanted to rediscuss that point, because it does not agree with the preprod.